src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -582,6 +601,7 @@ private static string[] GetDefaultLetterKeyHE(LetterKey letter)
                 LetterKey.VK_COMMA => new[] { "”", "’", "״", "׳" },
                 LetterKey.VK_PERIOD => new[] { "\u05ab", "\u05bd", "\u05bf" },
                 LetterKey.VK_MINUS => new[] { "–", "־" },
+                LetterKey.VK_QUOT_ => new[] { "”", "’", "'", },
```

Comment:
I think build is failing because of this mention to `VK_QUOT_`. Have you been able to build locally?

Reply:
I was able to build and test it. I'm going to look at my code if it's there too.

Reply:
Probably me who hit `ctrl + z` just before committing. Or a slip of attention.